CHANGELOG.md

```diff
@@ -7,9 +7,38 @@ For older changes see the [archived Singularity change log](https://github.com/a
 
 ## Changes Since Last Release
 
+### Bug fixes
+
+- Fix `GOCACHE` environment variable settings when building debian source
+  package on PPA build environment.
```

Comment:
Don't add a new bug fixes section.  Combine it with the existing one.

Reply:
Actually, in this branch we usually just use "Changes since last release".  I am fixing up the CHANGELOG in the release-1.1 branch in another PR so it's probably best to just wait for that and rebase on it.

Reply:
Alright, that's done.  Rebase on the current release-1.1 branch.

Reply:
I rebased to this branch to release-1.1.